README.md

```diff
@@ -1,135 +1,250 @@
+For more examples, see the [documentation](https://atlasgo.io/integrations/github-actions).
 
-#### `config-path`
+## API 
 
-Sets the path to the Atlas configuration file. By default, Atlas will
-look for a file named `atlas.hcl` in the current directory.
+### `ariga/setup-atlas`
 
-#### `config-env`
+Setup the Atlas CLI and optionally login to Atlas Cloud.
 
-Sets the environment to use from the Atlas configuration file. 
+#### Inputs
+* `cloud-token` - (Optional) The Atlas Cloud token to use for authentication.  To create
+   a cloud token see the [docs](https://atlasgo.io/cloud/bots).
+* `version` - (Optional) The version of the Atlas CLI to install.  Defaults to the latest
+   version.
 
-#### `dir`
+### `ariga/atlas-action/migrate/push` 
 
-Sets the directory that contains the migration scripts to analyze.
+Push the current version of your migration directory to Atlas Cloud.
 
-#### `dir-format`
+#### Inputs
 
-Sets the format of the migration directory. Options: `atlas` (default),
-`golang-migrate`, `goose`, `dbmate`, `flyway`, or `liquibase`.
+All inputs are optional as they may be specified in the Atlas configuration file.
 
-#### `dev-url`
+* `dir` - The URL of the migration directory to push.  For example: `file://migrations`.
+   Read more about [Atlas URLs](https://atlasgo.io/concepts/url).
+* `dir-name` - The name of the project in Atlas Cloud.  
+* `dev-url` - The URL of the dev-database to use for analysis.  For example: `mysql://root:pass@localhost:3306/dev`.
+   Read more about [dev-databases](https://atlasgo.io/concepts/dev-database).
+* `tag` - The tag to apply to the migration directory.  By default the current git commit hash is used.
+* `config` The path to the Atlas configuration file.  By default, Atlas will look for a file
+  named `atlas.hcl` in the current directory. For example, `file://config/atlas.hcl`.
+  Learn more about [Atlas configuration files](https://atlasgo.io/atlas-schema/projects).
+* `env` The environment to use from the Atlas configuration file.  For example, `dev`.
 
-The URL of the dev-database to use for analysis.
+#### Outputs
 
-* Read about [Atlas URL formats](https://atlasgo.io/concepts/url)
-* Read about [dev-databases](https://atlasgo.io/concepts/dev-database)
+* `url` - The URL of the migration directory in Atlas Cloud, containing an ERD visualization of the schema.
 
-#### `latest`
+### `ariga/atlas-action/migrate/lint`
 
-Use the `latest` mode to decide which files to analyze. By default,
-Atlas will use `git-base` to analyze any files that are present in the
-diff between the base branch and the current.
+Lint migration changes with Atlas
 
-Unless this option is set, the base branch (`master`/`main`/etc) must
-be checked out locally or you will see an error such as:
-```
-Atlas failed with code 1: Error: git diff: exit status 128
-```
+#### Inputs
+
+All inputs are optional as they may be specified in the Atlas configuration file.
```

Comment:
I think that the sir-name parameter is not optional, it can't be specified in the config file.

Reply:
It can by using remote_dir data source, right?

Reply:
You can define the data source, but there is no way to get the command to use the data source